frontend/src/components/common/EditorDialog.tsx

```diff
@@ -333,13 +344,10 @@ function DocsViewer(props: {docSpecs: any}) {
   }
 
   return (
-    docSpecs.error ?
-      <Empty>
-        Error getting documentation!
-        Please make sure the YAML is valid and has the kind and apiVersion set.
-      </Empty>
-      : docs === null ?
-        <Loader />
+    docs === null && docsError === null ?
+      <Loader />
+      : !_.isEmpty(docsError) ?
```

Comment:
Nitpick: You can easily just check for the `docsErorr` without using the `isEmpty` (as will only be valid if it's a non-empty string here).
But you don't really have to change this, it's just for future reference.